manim/animation/animation.py

```diff
@@ -275,8 +275,12 @@ def interpolate(self, alpha: float) -> None:
             The relative time to set the aniamtion to, 0 meaning the start, 1 meaning
             the end.
         """
-        alpha = min(max(alpha, 0), 1)
-        self.interpolate_mobject(self.rate_func(alpha))
+        # alpha = min(max(alpha, 0), 1)
+        self.interpolate_mobject(alpha)
+
+    @deprecated(until="v0.6.0", replacement="interpolate")
```

Comment:
We're already on version 0.7.0 so this deprecated decorator should be sometime after the current version (e.g. v0.8.0)